### PR TITLE
Todo doc cleanup

### PR DESCRIPTION
**Pull request by michael-basil**
**How to categorize this PR?**

/area documentation
/kind cleanup

**What this PR does / why we need it**:

Reduce confusion around the differentiation between "role" and "roles" field changes related to v1 api.

**Which issue(s) this PR fixes**:
Fixes # N/A

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

**Comment by gardener-robot-ci-1** [comment]
Thank you @MichaelRBasil for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

**Review by vpnachev — DISMISSED** [review]
/lgtm
/reviewed ok-to-test
/merge squash

File: pkg/apis/core/v1beta1/types_project.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ type ProjectMember struct {
 	// Role represents the role of this member.
 	// IMPORTANT: Be aware that this field will be removed in the `v1` version of this API in favor of the `roles`
 	// list.
-	// TODO: Remove this field in favor of the `owner` role in `v1`.
+	// TODO: Remove this field in favor of the `roles` list in `v1`..
```
Comment (ialidzhikov) [inline pkg/apis/core/v1beta1/types_project.go]:
```suggestion
	// TODO: Remove this field in favor of the `roles` field in `v1`.
```

**Comment by vpnachev** [comment]
@MichaelRBasil you need to run `make generate` and to contribute changes in the generated code.

/status author-action

File: pkg/apis/core/v1beta1/types_project.go (hunk 1 above)
Comment (michael-basil) [reply pkg/apis/core/v1beta1/types_project.go]: I pushed a commit to fix the typo.  I also ran `make generate` to be sure there were no downstream changes needed.

**Comment by michael-basil** [comment]
I pushed a commit to fix the typo. I also ran `make generate` to be sure there were no downstream changes needed.

**Review by vpnachev — APPROVED** [review]
/lgtm
/reviewed ok-to-test

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**